File: programmers/level_2/failed/가장 큰 수.py
def solution(numbers):
    numbers = list(map(str, numbers))
    # lambda 값이 numbers에 저장되는 것이 아닌 그냥 비교를 위해서 값을 부풀려주는 것일 뿐......
    numbers.sort(key=lambda x: x * 3, reverse=True)
    return str(int(''.join(numbers)))

File: programmers/고득점 Kit/sort/가장 큰 수.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string gptSolution(vector<int> numbers) {
    vector<string> strNum;
    
    for (const int& n: numbers) {
        strNum.push_back(to_string(n));
    }
    
    sort(strNum.begin(), strNum.end(), [](const string& a, const string& b) {
        return a+b > b+a;
    });
    
    string answer;
    for (const string& s: strNum) {
        answer += s;
    }
    
    return (strNum[0] == "0") ? "0" : answer;
}